qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -74,6 +76,82 @@ class EvaluationRule(Enum):
     DIAG = "diag"
 
 
+def _commutator(op_a: SparsePauliOp, op_b: SparsePauliOp) -> SparsePauliOp:
```

Comment:
I'm fine with both.

 For opflow we have had `commutator`. https://github.com/Qiskit/qiskit-terra/blob/2eee56616d50a9e26756f855ef4aa0135920ad78/qiskit/opflow/utils.py
It would be nice to have it for `LinearOp `. i.e. `commutator(op1': LinearOp, op2: LinearOp) -> LinearOp`

Reply:
Yes the opflow utility is exactly what we mimicked in Nature here: https://github.com/Qiskit/qiskit-nature/blob/ef1d99ecd735daa20e5603eb0b13fb9e25a614df/qiskit_nature/second_q/operators/commutators.py

I agree that it would be nice to have this directly for `LinearOp`. That would mean it should go somewhere into `quantum_info` :+1: I will open an issue shortly to track this over in Terra

Reply:
https://github.com/Qiskit/qiskit-terra/issues/9490